lms/djangoapps/badges/events/course_meta.py

```diff
@@ -0,0 +1,76 @@
+"""
+Events which have to do with a user doing something with more than one course, such
+as enrolling in a certain number, completing a certain number, or completing a specific set of courses.
+"""
+
+from badges.models import CourseEventBadgesConfiguration, BadgeClass
+from badges.utils import requires_badges_enabled
+
+
+def award_badge(config, count, user):
+    """
+    Given one of the configurations for enrollments or completions, award
+    the appropriate badge if one is configured.
+    """
+    slug = config.get(count)
+    if not slug:
+        return
+    badge_class = BadgeClass.get_badge_class(
+        slug=slug, issuing_component='edx__course', description=None, criteria=None,
+        image_file_handle=None, display_name=None, create=False,
+    )
+    if not badge_class:
+        return
+    if not badge_class.get_for_user(user):
+        badge_class.award(user)
+
+
+def enrollment_check(user):
+    """
+    Awards badges based on the number of courses a user is enrolled in.
```

Comment:
I certainly woudn't have guessed that from the name of the method, but I find it difficult to think of a better name without becoming Java-esque.